web/js/include/DateIntervalForm.js

```diff
@@ -229,6 +230,55 @@ Ext.ux.DateIntervalForm = Ext.extend(Ext.Panel, {
                         this._fireViewEvent();
                     }
                 }, '-', {
+                    text: '< Previous week',
+                    xtype: 'button',
+                    scope: this, //scope inside the handler will be the Form object
+                    handler: function () {
+                        var pivot = this._getStartDateField().getValue();
+                        if (!pivot)
+                            return;
```

Comment:
This doesn't patch this part of the specification of the prev/next week buttons: "If the date filters are empty, buttons would act with regard to the current date."